### PR TITLE
Pull Request - Fortifier42

### PR DESCRIPTION
**Pull request by Fortifier42**
Update Meta for Command

### CODE REVIEW

File: src/main/java/org/mcmonkey/dIRCBot/IRCCommand.java [hunk 1]
```diff
@@ -34,22 +34,25 @@
 
     // @Usage
     // Use to connect to an IRC server.
-    // - ~irc connect ircserver@irc.esper.net
+    // - irc connect ircserver@irc.esper.net
```
Comment (mcmonkey4eva) [inline src/main/java/org/mcmonkey/dIRCBot/IRCCommand.java]: Bad

File: src/main/java/org/mcmonkey/dIRCBot/IRCCommand.java [hunk 2]
```diff
@@ -34,22 +34,25 @@
...
     // @Usage
     // Use to connect to an IRC server.
-    // - ~irc connect ircserver@irc.esper.net
+    // - irc connect ircserver@irc.esper.net
     // - irc raw ircserver@irc.esper.net "USER dIRCBot mcmonkey.org mcmonkey.org dIRCBot"
     // - irc raw ircserver@irc.esper.net "NICK dIRCBot"
     // @Usage
     // Use to join a channel within an IRC server.
-    // - ~irc join ircchannel@irc.esper.net#denizen-dev
+    // - irc join ircchannel@irc.esper.net#denizen-dev
```
Comment (mcmonkey4eva) [inline src/main/java/org/mcmonkey/dIRCBot/IRCCommand.java]: Bad

Comment (Fortifier42) [inline src/main/java/org/mcmonkey/dIRCBot/IRCCommand.java]: why

Comment (mcmonkey4eva) [inline src/main/java/org/mcmonkey/dIRCBot/IRCCommand.java]: The `~` is needed to ensure the channel is joined before you send to it.

Comment (Fortifier42) [inline src/main/java/org/mcmonkey/dIRCBot/IRCCommand.java]: Okay, that makes sense.

**Pull request closed without merging**